labman/gui/test/test_pool.py

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-, labman development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+from unittest import main
+from tornado.escape import json_decode
+from labman.gui.testing import TestHandlerBase
+
+
+class TestPoolHandlers(TestHandlerBase):
+    def test_get_plate_list_handler(self):
+        response = self.get('/pool_list')
+        self.assertEqual(response.code, 200)
+        obs = json_decode(response.body)
+        self.assertCountEqual(obs.keys(), ['data'])
+        obs_data = obs['data']
+        self.assertEqual(len(obs_data), 3)
+        self.assertEqual(obs_data[0], [1, 'Test Pool from Plate 1', True, 1])
+        self.assertEqual(obs_data[1], [2, 'Test sequencing pool 1', False, 2])
```

Comment:
Just out of curiosity, why not test obs_data[2]?

Reply:
Just lazily following the approach of the code I copied from, which just tested the overall number of items and only a few of the individuals that were known to be interesting.  However, since there are only three items here, I went ahead and added the third test :)